pytui/backends/stackoverflow.py

```diff
@@ -67,5 +67,10 @@ def search(self, error_message: str, max_results: int = 5) -> List[StackOverflow
                         }
                     ).json(),
                 ])
+        print(answers[0][1])
```

Comment:
Can you remove this print statement? A lot of spam in the terminal

Reply:
Done